node/network/statement-distribution/src/lib.rs

```diff
@@ -2007,6 +2056,40 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			received_requests: prometheus::register(
+				prometheus::Counter::new(
+					"parachain_statement_distribution_received_requests_total",
+					"Number of large statement fetching requests received."
+				)?,
+				registry,
+			)?,
+			sent_responses: prometheus::register(
+				prometheus::CounterVec::new(
+					prometheus::Opts::new(
+						"parachain_statement_distribution_sent_responses_total",
+						"Number of served requests for large statement data."
+					),
+					&["success"],
+				)?,
+				registry,
+			)?,
+			sent_requests: prometheus::register(
+				prometheus::Counter::new(
+					"parachain_statement_distribution_sent_requests_total",
+					"Number of large statement fetching requests sent."
+				)?,
+				registry,
+			)?,
+			received_responses: prometheus::register(
+				prometheus::CounterVec::new(
+					prometheus::Opts::new(
+						"parachain_statement_distribution_received_responses_total",
+						"Number of received responses for large statement data."
+					),
+					&["success"],
+				)?,
+				registry,
+			)?,
```

Comment:
we have some metrics in substrate that are parameterized by protocol
https://github.com/paritytech/substrate/blob/b5c1c81235a27c04589360ea7c1bb525c9d2fde5/client/network/src/service/metrics.rs#L221-L257
are these more granular than that?

Reply:
Does not look like it - thanks!

Reply:
Well a bit, in case of the requester - will keep those.